.gitlab-ci.yml

```diff
@@ -27,44 +27,41 @@ variables:
 
 include:
   - project: "Northern.tech/Mender/mendertesting"
-    file: ".gitlab-ci-check-commits.yml"
-  - project: "Northern.tech/Mender/mendertesting"
-    file: ".gitlab-ci-github-status-updates.yml"
+    file:
+      - ".gitlab-ci-check-commits.yml"
+      - ".gitlab-ci-github-status-updates.yml"
+  - local: "/frontend/pipeline.yml"
 
 stages:
+  - pre-test
```

Comment:
What is the purpose of this stage, and why are all the frontend test jobs running in it?

Reply:
The frontend tests used to be in a test stage that precedes the build stage (same as commit checks etc.), since most run quicker - so the pipeline can be stopped earlier. Since the backend changed this to move the test stage to post-build, this is running the tests pre-build 🤷‍♂️

Reply:
You can use an empty `needs` to start a job immediately.

Reply:
or I can leave the jobs as they were and get the same behaviour with a smaller change 🤷‍♂️

Reply:
Up to you